tox.ini

```diff
@@ -17,6 +17,6 @@ deps =
 	pycodestyle
 	pylint
 commands =
-	pycodestyle calc/__init__.py calc/tests/__init__.py
-	pylint calc/tests/__init__.py
+	pycodestyle calc symmath
+	pylint tests
```

Comment:
@Carlos-Muniz Ideally, we should be running both pycodestyle and pylint on all three root directories (calc, symmath, and tests).

I don't know why we were only running these checks on specific files before... I think the original author of this `tox.ini` may have been confused.

Reply:
Ok, turns out these checks were intentionally only being run on specific files: https://github.com/openedx/openedx-calc/pull/1#issuecomment-485952972. The original extractor of `openedx-calc` from edx-platform didn't want to fix all the pylint and pycodestyle violations ;)

I am OK leaving the existing sins in place (ie, not running quality checks on the code that was already here), but I'd like for the new symmath code & tests you are adding to be totally style-checked and linted, as they were in edx-platform.

Reply:
I've run quality checks and removed all the lint.